public_suffix_list.dat

```diff
@@ -13304,6 +13304,10 @@ try-snowplow.com
 // Submitted by Drew DeVault <sir@cmpwn.com>
 srht.site
 
+// SparrowHost : https://sparrowhost.in/
+// Submitted by Anant Pandey <anant@sparrowhost.in>
```

Comment:
Please use a non-personal e-mail address.

Reply:
Hi there, ind.mom domain is currently in renewal state, Kindly allow sometime, it will be renewed for 2 years. i have updated the email, and added the _psl.ind.mom txt record, kindly check